src/Features/Core/Portable/InlineHints/InlineHintsOptions.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Options.Providers;
+
+namespace Microsoft.CodeAnalysis.InlineHints
+{
+    internal static class InlineHintsOptions
+    {
+        public static readonly PerLanguageOption2<bool> EnabledForParameters =
+            new(nameof(InlineHintsOptions),
+                nameof(EnabledForParameters),
+                defaultValue: false,
+                storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.InlineParameterNameHints.EnabledForParameters"));
```

Comment:
We should keep this specific RoamingProfileStorageLocation to be the same as the prior one in FeaturesOnOffOptions; otherwise this means anybody who has this feature turned on will get it turned back off for them once they get the upgraded bits. And yes, it's not consistent with the naming you otherwise have set up here, but consistency matters less then actually breaking users.

Reply:
Have pinged you offline about this.  This feature is currently explicitly marked as 'experiemental'.  so I think that that has given us permission to do this sort of thing.  tehre is an open question if we shoudl try to avoid the annoyance and how that plays into things.  IMO, the annoyance is fine, but if you feel strongly, i can keep the option unchanged. LMK.

Reply:
My comment to @CyrusNajmabadi privately was yes, it's experimental, but we shouldn't break users who opted in if we can do trivial work to avoid it.